__tests__/app.test.js

```diff
@@ -50,6 +40,24 @@ describe('/api/topics', () => {
   })
 });
 
+describe.only('/api/articles', () => {
+  test('Responds with the articles data and status 200', () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body}) => {
+        const {articles} = body;
+
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("created_at", {descending: true});
+        articles.forEach((article) => {
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.votes).toBe("number");
+        })
```

Comment:
This is shaping up to be a great test, I would like to see it be finished by adding an assertion for all the properties you are expecting. `title` and `votes` are just two of eight properties we would expect to exist on the object